server/index-node/src/schema.graphql

```diff
@@ -57,6 +59,21 @@ type EthereumIndexingStatus implements ChainIndexingStatus {
   lastHealthyBlock: Block
 }
 
+type EntityChanges {
+  updates: [EntityData!]!
+  deletes: [String!]!
+}
+
+type EntityData {
+  entityType: String!
+  fields: [EntityField!]!
+}
+
+type EntityField {
+  field: String!
```

Comment:
I'd prefer `name`

Reply:
Moved it all inside a `JSONObject`; marking as resolved.